charts/rekor/values.yaml

```diff
@@ -12,9 +12,10 @@ redis:
     - "yes"
   name: redis
   image:
-    repository: docker.io/redis
+    registry: docker.io
+    repository: redis
     pullPolicy: IfNotPresent
-    version: "5.0.10"
+    version: "sha256:0a0d563fd6fe5361316dd53f7f0a244656675054302567230e85eb114f683db4"
```

Comment:
don't we make a comment to say which tag is that? will be useful when updating the image. Or add a comment in how to discovery from which release is that.

same for others.

Reply:
@cpanato updated

Reply:
thanks!